src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -851,6 +851,91 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Status of the transparent data encryption.',
                    arg_type=get_enum_type(TransparentDataEncryptionStatus))
 
+    ###############################################
+    #                sql db ltr                   #
+    ###############################################
+    with self.argument_context('sql db ltr-policy set') as c:
+        create_args_for_complex_type(
+            c, 'parameters', Database, [
+                'weekly_retention',
+                'monthly_retention',
+                'yearly_retention',
+                'week_of_year'])
+
+        c.argument('weekly_retention',
+                   help='Retention for the weekly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
```

Comment:
Consider defining reusable param definition at top of file in order to avoid duplication between db ltr & mi ltr.

Reply:
Sure; can do once LTR MI is merged